docs/intelligentapps/bulkrun.md

```diff
@@ -0,0 +1,42 @@
+---
+Order: 4
+Area: intelligentapps
+TOCTitle: Bulk Run
+ContentId:
+PageTitle: Bulk Run Prompts
+DateApproved:
+MetaDescription: Run a set of prompts in an imported dataset, individually or in a full batch towards the selected genAI models and parameters.
+MetaSocialImage:
+---
+
+# Bulk Run Overview
+
+Bulk run feature in AI Toolkit allows you to run many prompts in batch mode. Compared to playground you can only run each prompt manually in order.
```

Comment:
```suggestion
The bulk run feature in AI Toolkit allows you to run multiple prompts in batch. When you use the playground, you can only run one prompt manually at a time, in the order they're listed.
```

Reply:
Add more information about how this works because in the next section you're mentioning a dataset. Explain briefly how the batch thing works.

Reply:
updated.